macros/insert_into_connected_components.sql

```diff
@@ -1,6 +1,6 @@
 {% macro insert_into_connected_components(loop_counter) %}
 {% set query %}
-insert into {{ ref('base_table') }} (
+insert into {{ ref('tasman_identity_base_table') }} (
```

Comment:
Q: do we need `tasman` in the table name?

Reply:
Tha package name itself makes sense, especially deploying this to dbt labs packages.
But the table names/usability could be focused on the task and not creator